packages/SwingSet/src/storeModule.js

```diff
@@ -11,6 +11,7 @@ export const {
 
 export {
   M,
+  defineHeapKind,
```

Comment:
Where does this `defineHeapKind` come from? Would we need to change swingset to import it from `store` and add it to the global `VatData`?

I'm hoping we can clean up the relationship between `store`, swingset, and downstream clients. Having ERTP depend upon swingset seems like a bad idea, but I'm not sure to to split the definition of the extended stores (their types, really), the implementation of the basic forms (which don't require swingset), the mock implementations of the more complex types (for unit tests), and the module that exposes the real implementation of the complex types (from swingset).

Reply:
I just cargo culted what we did for heap stores, which I like.

`@agoric/store` should not depend on swingset. The heap-only abstractions should not depend on swingset.